app/src/main/java/com/kodeco/android/countryinfo/ui/components/CountryInfoList.kt

```diff
@@ -31,9 +30,8 @@ fun CountryInfoList(
 ) {
     var selectedCountry: Country? by remember { mutableStateOf(null) }
 
-    // TODO: Replace these with standard `remember`ed objects that utilize `mutableStateOf()`
-    val tapCounter = Flows.tapFlow.collectAsState()
-    val backCounter = Flows.backFlow.collectAsState()
+    val tapCounter = remember { mutableStateOf(0) }
```

Comment:
Now that we're utilizing actual flows we could just remove the counter properties altogether. Up to you though!